content/articles/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Articles"
+date: 2018-09-13T15:10:47-07:00
```

Comment:
If possible could we add an OpenCensus logo in here? I ask because articles pages on their own can be a great copyable resource so it would be nice for the OpenCensus logo and brand to stand out on the articles landing page.